case/src/test/kotlin/com/ritense/case/web/rest/CaseTabManagementResourceIntTest.kt

```diff
@@ -218,4 +219,32 @@ class CaseTabManagementResourceIntTest @Autowired constructor(
             ).getOrNull()
         ).isNull()
     }
+
+    @Test
+    @Transactional
+    @WithMockUser(username = "admin@ritense.com", authorities = [ADMIN])
+    fun `should get tabs`() {
+        val caseDefinitionName = "my-case-type"
+
+        val key = "some-key"
+        val caseTab = CaseTab(
+            id = CaseTabId(caseDefinitionName, key),
+            name = "Some tab name",
+            type = CaseTabType.STANDARD,
+            tabOrder = Integer.MAX_VALUE,
+            contentKey = "some-content-key"
+        )
+
+        caseTabRepository.save(caseTab)
+
+        mockMvc.perform(
+            MockMvcRequestBuilders.get("/api/management/v1/case-definition/{caseDefinitionName}/tab", caseDefinitionName)
```

Comment:
```suggestion
            get("/api/management/v1/case-definition/{caseDefinitionName}/tab", caseDefinitionName)
```
If you do the change I mentioned above, you have to also do this.